src/components/Card.js

```diff
@@ -48,6 +80,10 @@ export default class Card {
     this._cardElement.querySelector(".cards__image").alt = this._name;
     this._cardElement.querySelector(".cards__title").textContent = this._name;
 
+    this._likeButton = this._cardElement.querySelector(".cards__like-button");
```

Comment:
You don't use `this._likeButton`. Please, delete it or use it all over the `Card`